platform/ui/src/components/SplitButtonToolbar/__stories__/splitButtonToolbar.stories.mdx

```diff
@@ -0,0 +1,80 @@
+import SplitButton from '../SplitButtonToolbar';
+
+import { ArgsTable, Story, Canvas, Meta } from '@storybook/addon-docs';
+import { createComponentTemplate } from '../../../storybook/functions/create-component-story';
+import ToolbarButton from '../../ToolbarButton';
+
+export const argTypes = {
+  component: SplitButton,
+  title: 'Components/SplitButtonToolbar',
+};
+
+<Meta
+  title="Components/SplitButtonToolbar"
+  component={SplitButton}
+/>
+
+export const SplitButtonTemplate = args => (
+  <div className="bg-primary-dark h-32">
+    <SplitButton {...args} />
+  </div>
+);
+
+<Heading
+  title="SplitButton"
+  componentRelativePath="SplitButtonToolbar/SplitButtonToolbar.tsx"
+/>
+
+- [Overview](#overview)
+- [Props](#props)
+- [Contribute](#contribute)
+
+## Overview
+
+SplitButton is a component that renders the SplitButtons.
```

Comment:
This is in splitBUttonToolbar - can you tweak the text to explain why it is splitButtonToolbar?